reporting/api/src/main/java/org/jboss/windup/reporting/model/ClassificationModel.java

```diff
@@ -97,4 +104,12 @@
     @SetInProperties(propertyPrefix = "tag")
     Set<String> getTags();
 
+    abstract class Impl implements ClassificationModel, JavaHandlerContext<Vertex>
```

Comment:
Why not just add the "transformToFileModel()" method to this interface and annotate it in here? It seems a little convolute that ToFileModelTrasformable has the annotation but not the implementation.